components/02-elements/button/button.config.js

```diff
@@ -30,6 +30,13 @@ module.exports = {
         attributes: 'href="#"',
         text: 'I am a link button',
       }
-    }
+    },
+    {
+      name: 'wide',
```

Comment:
changed to `fluid` IMO 'wide' for me is more explicite b/c even in small container button is 100% width ;) but I've changed.